docker-compose.yml

```diff
@@ -14,5 +14,6 @@ services:
         - DOCS_ENFORCE_GIT_LOG_HISTORY
       context: .
     image: docs/docstage
+    pull_policy: never
```

Comment:
We could change this to `build`, but then we would _alway_ force the image to be built; sometimes you may want to "up" without rebuilding.